rs/ethereum/ledger-suite-orchestrator/src/state/mod.rs

```diff
@@ -213,7 +215,100 @@ pub type GitCommitHash = Hash<GIT_COMMIT_HASH_LENGTH>;
 
 #[derive(Clone, PartialEq, Debug, Default, Deserialize, Serialize)]
 pub struct ManagedCanisters {
+    /// Canisters for an ERC-20 token
     canisters: BTreeMap<Erc20Token, Canisters>,
+    /// Canisters for non ERC-20 token, identified by their token symbol
+    #[serde(default)]
+    other_canisters: BTreeMap<TokenSymbol, Canisters>,
```

Comment:
This I thought would have been more tricky, because `ManagedCanisters` is already stored in stable memory. So to do the change you suggested, I would need to upgrade the stored bytes in stable memory during the `post_upgrade`. In contrast adding a new field (with `#[serde(default)]`) should work right out-of-the box. Does-it make sense?

Reply:
I see. So it is for compatibility reasons. Thanks for the explanation!